src/update_handler.c

```diff
@@ -2527,6 +2532,102 @@ static gboolean check_if_area_is_clear(const gchar *device, guint64 start, gsize
 	return res;
 }
 
+static gboolean img_to_boot_nor_fallback_handler(RaucImage *image, RaucSlot *dest_slot, const gchar *hook_name, GError **error)
+{
+	GError *ierror = NULL;
+	gboolean res = FALSE;
+	gchar** devices = NULL;
+	gboolean primary_clear;
+	char *part_names[2] = {
+		"primary",
+		"fallback"
+	};
+	int first_part_desc_index = 1;
+	const gsize header_size = 512; /* TODO: this value should be settable through the system config file. */
```

Comment:
Yes, this needs to be configurable.

My suggestion would be a `header-size` field in the system config and a `guint64 header_size` in `RaucSlot`, similar to how region_size works.